packages/browser/src/sessionInfo/SessionInfoManager.ts

```diff
@@ -59,13 +60,14 @@ export async function clear(
     // FIXME: This is needed until the DPoP key is stored safely
     storage.delete("clientKey", { secure: false }),
   ]);
+  await clearOidcPersistentStorage();
```

Comment:
Won't this clear **_all_** `oidc-client-js` state, yet this method is supposed to be session-specific? Anyway, there are bigger problems right now with this whole area...

Reply:
Given that `oidc-client-js` is browser-specific, I reckon they don't expect multiple sessions to coexist in the same browser, so I guess they don't really care about making the distinction.